#goal definition
geometry_msgs/PoseStamped origin
float32 vertical_distance
float32 horizontal_distance
float32 horizontal_velocity  # positive values indicates ccw, negative cw
uint8 revolutions_count
---
#result definition
string message
---
#feedback
geometry_msgs/PoseStamped current_target
